proxy/src/main/java/net/md_5/bungee/http/HttpClient.java

```diff
@@ -27,7 +27,7 @@
 {
 
     public static final int TIMEOUT = 5000;
-    private static final Cache<String, InetAddress> addressCache = CacheBuilder.newBuilder().expireAfterWrite( 5, TimeUnit.MINUTES ).build();
+    private static final Cache<String, InetAddress> addressCache = CacheBuilder.newBuilder().expireAfterWrite(1, TimeUnit.MINUTES).build();
```

Comment:
You should however leave the format, its the bungeecord code style having spaces there.